.github/workflows/test.yml

```diff
@@ -19,12 +19,8 @@ jobs:
         go-version: ${{ matrix.go-version }}
     - name: Checkout code
       uses: actions/checkout@v2
-      with:
```

Comment:
This needs to be kept for compatibility with lower versions of Go

Reply:
It's great without it.

Reply:
Oh, I see, the manual setup of dependencies were necessary folder older versions of Go before module support.